requirements.txt

```diff
@@ -1,44 +1,131 @@
-django==2.2.10
-gunicorn==19.9.0
-psycopg2-binary==2.7.6.1
-pytz==2018.9
-whitenoise[brotli]==4.1.2
-requests==2.21.0
-ipdb==0.11
-geographiclib==1.49
-geopy==1.18.1
-git+git://github.com/rtcharity/django-environ@develop#egg=django-environ
-git+git://github.com/rtcharity/django-authtools@master#egg=django-authtools
-django-storages[azure]==1.8
-django-redis==4.10.0
-hiredis==1.0.0
-Pillow==6.2.0
-sorl-thumbnail==12.5.0
-django-cleanup==3.0.1
-django-upload-path==0.1.1
-airtable-python-wrapper==0.11.3.post1
-applicationinsights==0.11.7
-django-autoslug==1.9.4
-django-enumfield==1.5
-django-crispy-forms>=1.7.2
-rules==2.0.1
-pytils==0.3
-mysqlclient==1.4.2.post1
-django-sluggable==0.6.1
-django-allauth==0.39.1
-django-recaptcha==2.0.2
-django-pwned-passwords==3.0.0
-jasmine==3.4.0
-pytest==4.3.0
-pytest-django==3.4.8
-pytest-selenium==1.16.0
-django-referrer-policy==1.0
-django-feature-policy==2.3.0
-black==19.3b0
-pytest-black==0.3.7
-pytest-flake8==1.0.4
-pep8-naming==0.8.2
-isort[requirements]==4.3.20
-flake8-isort==2.7.0
-django-webpack-loader==0.6.0
-django-admin-relation-links==0.2.4
+#
```

Comment:
Why would you like to implement this change? Is this necessary for sorting the webpack stuff?
If not, I'd suggest to keep the PR as small as possible to achieve its main goal.

Reply:
Hi @sebgrebe, regarding the necessity - I would say yes and no. In general I wouldn't advice to run a production server without compiled requirements, it isn't safe for a number of reason - I wouldn't like to describe them in text, but we can talk about it if you wish :)

Reply:
Let's do that! But let's make this change part of a different PR, to keep PRs manageable